docfx_yaml/extension.py

```diff
@@ -396,11 +396,86 @@ def indent_code_left(lines, tab_space):
     return "\n".join(parts)
 
 
-def _parse_docstring_summary(summary):
+def _parse_enum_content(parts: Sequence[str]) -> Sequence[Mapping[str, str]]:
+    """Parses the given content for enums.
+
+    Args:
+        parts: The content to parse, given in the form of separated by
```

Comment:
`in the form of a sequence of str, which have been split by newlines and are left-indented.`